frontend/src/booking/calendar.vue

```diff
@@ -104,6 +104,11 @@
       this.$root.$on('toggleOffsite', (bool) => { this.toggleOffsite(bool) })
       this.$root.$on('unselect', () => { this.unselect() })
       this.$root.$on('updateEvent', (event, params) => { this.updateEvent(event, params) })
+      document.addEventListener('keydown', e => {
```

Comment:
This change adds the event listener to the document, should it go into App.js, or somewhere? It looks like if you navigate between the calendar and opening the add-exam-modal multiple times, this event is going to be registered over and over again.

Reply:
Fixed, tested and pushed up.